flytekit/clis/sdk_in_container/register.py

```diff
@@ -148,14 +172,27 @@ def register(
     """
     see help
     """
+
```

Comment:
Same here can we prevent users from passing in `--non-fast` and `--copy` at the same time? (We do not check if the values are consistent)

Reply:
added.